src/coreclr/interpreter/compileropt.cpp

```diff
@@ -375,7 +387,7 @@ void InterpCompiler::AllocOffsets()
                 {
                     int32_t var = m_pActiveVars->Get(i);
                     if (m_pVars[var].alive)
-                        printf(" %d (end %d),", var, m_pVars[var].liveEnd);
+                        printf(" %d (end %p),", var, m_pVars[var].liveEnd);
```

Comment:
Printing a pointer doesn't seem really helpful. Not sure what would be best, something specific and compact. Maybe, for now, we could print ilOffset and the opcode name instead ?